File: library/src/main/java/com/jess/arms/utils/AutoLayout.java
package com.jess.arms.utils;

import android.view.View;
import android.view.ViewGroup;

import com.zhy.autolayout.utils.AutoUtils;

/**
 * Created by J1aDong on 2017/1/19.
 */

public class AutoLayout
{

	public static void auto(View view)
	{
		if( view instanceof ViewGroup )
		{
			for( int i = 0; i < ((ViewGroup) view).getChildCount(); i++ )
			{
				AutoUtils.auto(view);
				auto(((ViewGroup) view).getChildAt(i));
			}
		}else {
            AutoUtils.auto(view);
        }
	}
}

File: app/src/main/java/com/j1adong/huabankotlin/AutoLayout.kt
package com.j1adong.huabankotlin

import android.view.View
import android.view.ViewGroup
import com.zhy.autolayout.utils.AutoUtils

/**
 * Created by J1aDong on 2017/1/6.
 */
class AutoLayout {
    companion object {
        fun auto(view: View): View {
            if (view is ViewGroup) {
                for (i in 0..view.childCount - 1) {
                    auto(view.getChildAt(i))
                }
            }
            AutoUtils.auto(view)
            return view
        }
    }
}